### DISCUSSION TITLE
Using RTD Version selector while not hosting on RTD?

### DISCUSSION THREAD

**Discussion by AVHopp** [pradyunsg/furo#814] [Questions] [answered]
Hey everyone, I have a (potentially stupid?) question: Is it possible to use the ReadTheDocs Version Selector without the documentation being hosted on RTD?
I would like to create a look-and-feel similar to attrs (https://www.attrs.org/en/stable/), but we do not want to host our documentation on RTD. Also, I have seen that there are *some* things related to RTD in the code, but I honestly did not manage to understand whether this is just for using `furo` on a RTD hosted documentation or can be used for my purpose.
Thank you 😄

**Comment by pradyunsg** [comment, marked as answer]
It is not currently feasible sadly, no.

**Reply by AVHopp** [reply]
That was also my impression, so thanks for confirming 👍
